packages/kbn-profiling-utils/common/frame_group.ts

```diff
@@ -44,5 +61,9 @@ export function createFrameGroupID(
     return `elf;${fileID};${functionName}`;
   }
 
+  if (equivalenceMap.has(`${sourceFilename}:${functionName}`)) {
+    var replacementString = equivalenceMap.get(sourceFilename + ":" + functionName);
```

Comment:
I named the variable as `key` but you can give it a better name.
```suggestion
  const key = `${sourceFilename}:${functionName}`
  if (equivalenceMap.has(key)) {
    var replacementString = equivalenceMap.get(`${sourceFilename}:${functionName}`);
```